modules/level_set/python/lsutils.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
Hmm, might be a first for a python directory under a module? Does this really go here? If so should it have headers?